### PR TITLE
IIIF Image API emulated in OpenAPI

### PR DESCRIPTION
**Pull request by VictorHarbo**

### CODE REVIEW

File: src/main/openapi/ds-image-openapi_v1.yaml [hunk 1]
```diff
@@ -273,14 +273,335 @@ paths:
...
+                    context:
+                      type: string
+                      description: URI of image
+                    id:
+                      type: string
+                      description:   The base URI of the image as defined in URI Syntax, including scheme, server, prefix and identifier without a trailing slash.
+                    type:
+                      type: string
+                      description: The type for the Image API. The value must be the string ImageService3.
+                    protocol:
```
Comment (tokee) [inline src/main/openapi/ds-image-openapi_v1.yaml:451]: It would be even better with an example value for each element.

**Review by tokee — CHANGES_REQUESTED** [review]
Technically the main issue is that we need path instead of query params for IIIF.

The rest of my comments are more about style.

File: src/main/openapi/ds-image-openapi_v1.yaml [hunk 2]
```diff
@@ -273,14 +273,335 @@ paths:
...
+          type: string
+
+      x-streamingOutput: true
+      responses:
+        # TODO: Update description to explain errors from CVT, WID, HEI, RNG and where ever needed. PFL! CTW!
+        '200':
+            description: 'Succes!'
+            content: 
+              application/ld+json:
+                schema:
```
[the remark this answers is not in the document]
Comment (VictorHarbo) [reply src/main/openapi/ds-image-openapi_v1.yaml:438]: The schema has been moved to components and referenced. Github hasn't outdated it.

File: src/main/openapi/ds-image-openapi_v1.yaml [hunk 3]
```diff
@@ -27,7 +27,7 @@ paths:
   /:
     get:
       tags:
-        - 'ds-image'
+        - 'Access'
       summary: 'Internet Imaging Protocol available through OpenAPI specification. The specification can be found at: https://iipimage.sourceforge.io/documentation/protocol/'
```
Comment (tokee) [inline src/main/openapi/ds-image-openapi_v1.yaml:31]: "Imaging", not "Imageing". And maybe shorten the description like the IIIF one?

File: src/main/openapi/ds-image-openapi_v1.yaml [hunk 4]
```diff
@@ -280,11 +280,11 @@ paths:
...
 # IIIF Image request
-  /{identifier}/:
+  /{identifier}/{region}/{size}/{rotation}/{quality}.{format}:
     get:
       tags:
-        - 'IIF Image request'
-      summary: 'OpenAPI definition for IIIF Image Request API '
+        - 'Access'
+      summary: 'IIIF Image Request'
       operationId: IIIFImageRequest
```
Comment (tokee) [inline src/main/openapi/ds-image-openapi_v1.yaml:288]: The description says "Special characters must be URI encoded.". Maybe add a ", such as / and ?" and mention that the OpenAPI GUI will do it automatically if that is used.

File: src/main/openapi/ds-image-openapi_v1.yaml [hunk 5]
```diff
@@ -316,7 +316,7 @@ paths:
             type: string
```
Comment (tokee) [inline src/main/openapi/ds-image-openapi_v1.yaml:316]: It's best that there is no default, but samples would be nice. But I think that combination is not possible?

File: src/main/openapi/ds-image-openapi_v1.yaml [hunk 6]
```diff
@@ -360,7 +360,7 @@ paths:
...
```
Comment (tokee) [inline src/main/openapi/ds-image-openapi_v1.yaml:360]: Rotation could definitely use a default of 0 and some examples where at least one is not 0, 90, 180 or 270.

**Review by tokee — CHANGES_REQUESTED** [review]
Nearly there, just a few small things.

File: src/main/openapi/ds-image-openapi_v1.yaml (hunk 5 above)
Comment (VictorHarbo) [reply src/main/openapi/ds-image-openapi_v1.yaml:316]: I don't see the posibility to add examples without OpenAPI adding it to the result. I have written some examples in the textual description.

File: src/main/openapi/ds-image-openapi_v1.yaml (hunk 4 above)
Comment (VictorHarbo) [reply src/main/openapi/ds-image-openapi_v1.yaml:288]: Description is now updated

File: src/main/openapi/ds-image-openapi_v1.yaml (hunk 6 above)
Comment (VictorHarbo) [reply src/main/openapi/ds-image-openapi_v1.yaml:364]: Examples now added

**Review by tokee — APPROVED** [review]

**Pull request merged**